src/Symfony/Component/Translation/Tests/TranslatorCacheTest.php

```diff
@@ -59,6 +59,10 @@ protected function deleteTmpDir()
      */
     public function testThatACacheIsUsed($debug)
     {
+        if (!class_exists(\MessageFormatter::class)) {
+            $this->markTestSkipped(sprintf('Skipping test as the "%s" class does not exist.', \MessageFormatter::class));
```

Comment:
The class does not exist when `ext-intl` is missing. I think the message should reflect that.

Reply:
Good idea, I have updated the message to provide more information.

Reply:
Suggesting the PHP extension or the polyfill package is inline with the exception that is thrown by the `IntlFormatter`.